guides/common/assembly_configuring-external-idm-dns.adoc

```diff
@@ -10,7 +10,7 @@ The hosts are not involved in this process.
 Therefore, you must install and configure the IdM client on the {Project} or {SmartProxy} that is currently configured to provide a DNS service for the domain you want to manage using the IdM server.
 
 {ProductName} can be configured to use a Red{nbsp}Hat Identity Management (IdM) server to provide DNS service.
-For more information about Red{nbsp}Hat Identity Management, see the https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Linux_Domain_Identity_Authentication_and_Policy_Guide/[Linux Domain Identity, Authentication, and Policy Guide].
+For more information about Red{nbsp}Hat Identity Management, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/linux_domain_identity_authentication_and_policy_guide/index[Linux Domain Identity, Authentication, and Policy Guide].
```

Comment:
For other links, you omit the `/index` at the end. Intentional?

Reply:
not in the document, only the url, e.g. inconsequential